### ISSUE TITLE
Switching to Cookie Store

### ISSUE THREAD

**Issue by epyx25** [simplabs/ember-simple-auth#1749]
I have been using  ember simple auth since version 1.1.0 without any specified store (so Adaptive Store).
It works fine, I can see in my browser's Local Storage

key: **ember_simple_auth:session**
```Javascript
{
  authenticator: "authenticator:custom",
  id: "xxx",
  ttl: 1209600,
  userId: "---"
}
```
The problem: I want to terminate the session, if the user closes the browser. If I understand correctly, I have to use Cookie Storage, which will be emptied on closing the browser.

I created a file **app/session-stores/application.js**
```Javascript
import Cookie from 'ember-simple-auth/session-stores/cookie';

export default Cookie.extend();
```

But it does not to seem to change anything. There is no new entry among the cookies, but I can still see the `ember_simple_auth:session` key in the Local Storage, and the session survives if I close the browser.

I have _"ember-simple-auth": "1.1.0"_ but according to the old [documentation](https://github.com/simplabs/ember-simple-auth/tree/1.1.0#session-stores)  it should not be a problem.

**Comment by pichfl** [comment]
Just like the localStorage, the cookie store is also extend the session lifetime beyond the current session. To get a session that only lives until the current window is closed you can use the [Ephemeral Session Store](http://ember-simple-auth.com/api/classes/EphemeralStore.html)

The key in localStorage will not be removed if you switch to another session storage option, but it shouldn't be used either. 

Make sure your browser cache and stored data was completely cleared and restart your ember server.

**Comment by epyx25** [comment]
@pichfl thanks for your answer! 
I gave a try to the Ephemeral Session store, but it forgets the session on page reload. If a logged in user reloads the page, he / she ends up on the login page. 

So I made a second try with the Cookie Store. According to the [cookie store documentation](http://ember-simple-auth.com/api/classes/CookieStore.html):

> By default the cookie session store uses a session cookie that expires and is deleted when the browser is closed. The cookie expiration period can be configured by setting the cookieExpirationTime property.

After setting back the file **app/session-stores/application.js** and restart my **ember s** (not so sure why it is necessary) now I can see the following cookie.

![image](https://user-images.githubusercontent.com/17404207/54292365-5be6f700-45ae-11e9-87e4-aa706e8f38e6.png)

But after logging in, and closing / opening the browser I am still logged in, and I can see the same cookie entry.
